src/app/page.tsx

```diff
@@ -53,7 +59,7 @@ export default function HomePage() {
         <h1>Your shares. Now private.</h1>
         <h2>Scale your cap table with confidence.</h2>
         <p className="mt-4 text-lg sm:mt-8">
-          For the builders and funders. We&apos;ll never use your data to sell
+          For the builders and founders. We&apos;ll never use your data to sell to
```

Comment:
Woops, not typos - could you roll these back?
- Funder as in investor
- Secondaries as in "sell secondary shares"